samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -35,6 +35,14 @@ LOG_MODULE_REGISTER(at_host, CONFIG_SLM_LOG_LEVEL);
 #define AT_CMD_CLAC	"AT#XCLAC"
 
 #define AT_MAX_CMD_LEN		CONFIG_AT_CMD_RESPONSE_MAX_LEN
+#define UART_RX_LEN	8  /* UART FIFO depth is 6 (?) */
+
+/*
+ * Known limitation in this version
```

Comment:
I feel like this information is very hidden in the .c file and should be moved to a Known Limitations section in the .rst file.

Reply:
Will move them out of .c to .rst